Makefile

```diff
@@ -1,4 +1,4 @@
-.PHONY: all build build-debug clean test coverage lint vet fmt deps image push run
+.PHONY: all build build-debug clean test coverage lint vet fmt deps image push run deploy undeploy cluster-restart cluster-down cluster-up gen-metrics-docs
```

Comment:
Instead of this can we please have multiple .PHONY close to the target so that when you are deleting a target, the .PHONY gets removed as well?